### PR TITLE
Record ASM vnet vm_images cloud_service & locations integration tests

### PR DESCRIPTION
**Pull request by vishrutshah**
This changes is part of the feature #291

### CODE REVIEW

File: service_management/azure/test/integration/cloud_service/cloud_create_test.rb [hunk 1]
```diff
@@ -15,29 +15,30 @@
...
+    VCR.insert_cassette "cloud_service/#{name}"
   end
 
-  describe '#create_cloud_service' do
-    before do
-      @cloud_name = random_string('test-service-cloud', 10)
-      subject.create_cloud_service(@cloud_name, options)
-    end
+  after do
+    subject.delete_cloud_service(cloud_service_name)
```
Comment (veronicagg) [inline service_management/azure/test/integration/cloud_service/cloud_create_test.rb]: Should we have delete and creation in the same place? Either both in the test or create in the "before" and delete in "after"?

File: service_management/azure/test/integration/cloud_service/cloud_delete_test.rb [hunk 2]
```diff
@@ -15,30 +15,29 @@
...
-    end
+  after do
+    VCR.eject_cassette
+  end
 
-    it 'Deletes the cloud service in Microsoft Azure.' do
-      subject.delete_cloud_service(@cloud_name)
-      cloud_service = subject.get_cloud_service(@cloud_name)
+  describe 'cloud service' do
+    it 'should delete the cloud service' do
```
Comment (veronicagg) [inline service_management/azure/test/integration/cloud_service/cloud_delete_test.rb]: Should have one file containing create and delete cloud service tests? We end up doing almost the same in each file.

File: service_management/azure/test/integration/test_helper.rb [hunk 3]
```diff
@@ -1,77 +1,77 @@
-#-------------------------------------------------------------------------
```
Comment (veronicagg) [inline service_management/azure/test/integration/test_helper.rb]: What changed in this file?

File: service_management/azure/test/integration/cloud_service/cloud_create_test.rb [hunk 4]
```diff
@@ -15,29 +15,30 @@
...
 describe Azure::CloudServiceManagementService do
-  include Azure::Core::Utility
-
   subject { Azure::CloudServiceManagementService.new }
+  let(:cloud_service_name) { 'testcloudservice2375' }
   let(:options) do
     {
-      location: 'West US',
-      description: 'Test'
+      location: 'East US',
```
Comment (veronicagg) [inline service_management/azure/test/integration/cloud_service/cloud_create_test.rb]: Why do we need to update the location?

**Comment by veronicagg** [comment]
@vishrutshah Made some comments, rest looks good to me. Thanks!

File: service_management/azure/test/integration/cloud_service/cloud_delete_test.rb (hunk 2 above)
Comment (vishrutshah) [inline service_management/azure/test/integration/cloud_service/cloud_delete_test.rb]: Yes that's right. Merging both create & delete test into one file.

File: service_management/azure/test/integration/cloud_service/cloud_create_test.rb (hunk 4 above)
Comment (vishrutshah) [inline service_management/azure/test/integration/cloud_service/cloud_create_test.rb]: The test was failing that the location was not supported so i updated the location.

File: service_management/azure/test/integration/cloud_service/cloud_create_test.rb (hunk 1 above)
Comment (vishrutshah) [inline service_management/azure/test/integration/cloud_service/cloud_create_test.rb]: yup merging both create + delete test will eliminate this kind of usage.

File: service_management/azure/test/integration/test_helper.rb (hunk 3 above)
Comment (vishrutshah) [inline service_management/azure/test/integration/test_helper.rb]: actually there is nothing changed. Re-checked out this file from master.

**Pull request merged**